doc/contributing.md

```diff
@@ -97,7 +97,7 @@ where you can submit general comments and/or questions:
 ### Resources for New Contributors
```

Comment:
Please fix a typo (`Sow` => `Show`) at line 93. 
Changing the URL to `https://forum.generic-mapping-tools.org/c/Show-your-nice-example-script/10`.